src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Functions/ProcessInitialDataSetVersionFunction.cs

```diff
@@ -54,59 +48,83 @@ public class ProcessInitialDataSetVersionFunction(PublicDataDbContext publicData
                 context.InstanceId,
                 input.DataSetVersionId);
 
-            await context.CallActivityAsync(nameof(HandleProcessingFailure), input.DataSetVersionId);
+            await context.CallActivity(nameof(HandleProcessingFailure), logger);
         }
     }
 
+    [Function(nameof(ImportMetadata))]
+    public async Task ImportMetadata(
+#pragma warning disable IDE0060
+        [ActivityTrigger] object? input,
+#pragma warning restore IDE0060
```

Comment:
As mentioned in Slack, the pragmas are pretty horrible. Perhaps we should switch back to using the `instanceId` as the activity trigger?

Same applies for the other functions in this PR